Tests/ApolloServerIntegrationTests/StarWarsApolloSchemaDownloaderTests.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  StarWarsApolloSchemaDownloaderTests.swift
+//  ApolloServerIntegrationTests
+//
+//  Created by Anthony Miller on 4/20/21.
+//  Copyright © 2021 Apollo GraphQL. All rights reserved.
+//
+
+#if os(macOS)
+import XCTest
+import ApolloTestSupport
+@testable import ApolloCodegenLib
+
+class StarWarsApolloSchemaDownloaderTests: XCTestCase {
```

Comment:
I think this could be `ApolloSchemaDownloaderIntegrationTests` since there's nothing that's particularly required about the schema being downloaded being the SW schema, but not a hill I wanna die on.

Reply:
Oh, yeah, just all the other tests that depend on the SW server currently have StarWars at the start of the name.